Azure/azure-cli
# [Network] `az network virtual-appliance`: Add parameter `--additional-nics`

**Related command**


Network

**Description**


Added support of additional nic in az network virtual-appliance create

**Testing Guide**

//Create Command
az network virtual-appliance create -g RahulTestSetup -n RahulCheckPointNva4  --vhub "/subscriptions/62364504-2406-418e-971c-05822ff72fad/resourceGroups/RahulTestSetup/providers/Microsoft.Network/virtualHubs/RahulTestHub" --vendor "checkpoint" --scale-unit 10 -v latest --asn 64512 --init-config "" --additional-nics "[{name:sdwan,has-public-ip:true}]" --debug

![image](https://github.com/Azure/azure-cli/assets/108508612/dc05f5a9-b32a-4f6c-9543-5cedabd86f2d)


//Help
az network virtual-appliance create --additional-nics ??

Argument
    --additional-nics : List<Object>
        Details required for Additional Network Interface.
        See https://github.com/Azure/azure-cli/tree/dev/doc/shorthand_syntax.md for more about
        shorthand syntax.

List Element Properties
    has-public-ip : Flag (true or false) for Intent for Public Ip on additional nic  Allowed values:
                   0, 1, f, false, n, no, t, true, y, yes.
    name          : Name of additional nic


**History Notes**


[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [* ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [*] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).
